src/model_signing/_signing/sign_certificate.py

```diff
@@ -106,7 +109,10 @@ def __init__(
               signature and establish chain of trust. By default this is empty,
               in which case we would use the root certificates from the
               operating system, as per `certifi.where()`.
+            fingerprints: Display the fingerprints of certificates
         """
+        self.fingerprints = fingerprints
```

Comment:
I think this can be private to the class, so `self._log_fingerprints` or similar?